packages/workbox-webpack-plugin/index.js

```diff
@@ -62,6 +62,25 @@ class WorkboxBuildWebpackPlugin {
       config.globPatterns = ['**/*.{html,js,css}'];
     }
 
+    if (compilation.options.output.publicPath && !config.manifestTransforms) {
+      const publicPath = compilation.options.output.publicPath;
+      const compiledAssets = [];
+      for (let key in compilation.assets) {
+        if (compilation.assets.hasOwnProperty(key)) {
+          compiledAssets.push(path.resolve(publicPath, './' + key));
+        }
+      }
+      config.manifestTransforms= [(manifestEntries) =>
+          manifestEntries.map((entry) => {
+            if (compiledAssets.indexOf(path.resolve(publicPath, entry.url))
```

Comment:
Is this redundant? Doesn't checking for `compilation.assets[entry.url]` cover this?
I don't think you need the `compiledAssets` array, or do you?